scene/gui/grid_container.h

```diff
@@ -46,6 +47,8 @@ class GridContainer : public Container {
 public:
 	void set_columns(int p_columns);
 	int get_columns() const;
+	void set_expand_columns(bool p_expand);
+	bool get_expand_columns() const;
```

Comment:
Should be `is_expanding_columns` (well as mentioned before, this is not done consistently, but it still seems to be the dominant style for boolean getters).